calico/getting-started/kubernetes/self-managed-onprem/onpremises.mdx

```diff
@@ -59,6 +59,12 @@ installed directly on the cluster as a Deployment, and is configured through one
    ```
    kubectl create -f custom-resources.yaml
    ```
+1. Verify {{prodname}} installation in your cluster.
+
+   ```
+   watch kubectl get pods -n calico-system
+   ```
+1. (Optional) Verify that the pods can talk to each other and that the network policy works. For more information, see [Kubernetes policy](../../../network-policy/get-started/kubernetes-policy/kubernetes-policy-basic.mdx).
```

Comment:
Why is this one optional? 

Do we really want to assume that people can figure this out on their own? How do they check that the network policy works? And while we're at it, what network policy have we installed?

Reply:
The procedure is for installing Calico. When you install, you should verify if the installation works. Nothing beyond that. Checking the Kubernetes network policy does not seem to be part of the install workflow.

Reply:
If you want to include a verification step, then you should include information on how to perform the verification. If checking the network policy is not part of the workflow, then why include it here? Right now this seems to be uncomfortably in between.

Reply:
Optional step is removed.